package.json

```diff
@@ -44,6 +44,7 @@
     "ignore": "~5.1.4",
     "js-yaml": "~3.13.1",
     "jsonc-parser": "~2.2.0",
+    "junit-report-builder": "^2.0.0",
```

Comment:
‘~’, please

Reply:
This is what I get for just blindly copy/pasting from the internet, updated 😄